static/bidder-info/missena.yaml

```diff
@@ -0,0 +1,16 @@
+endpoint: https://bid.missena.io/
+maintainer:
+  email: yboussafa@missena.com
```

Comment:
this looks like a private email, please enter maintenance email

Reply:
Done ✅